File: PRAK401-2310817220016-NailaHanifah.c
#include <stdio.h>

int main() {
    int kelipatan;
    char simbol;

    printf("Input pertama (bilangan kelipatan): ");
    scanf("%d", &kelipatan);
    printf("Input kedua (simbol pengganti): ");
    scanf(" %c", &simbol);

    for (int i = 1; i <= 50; i++) {
        if (i % kelipatan == 0) {
            printf("%c ", simbol);
        } else {
            printf("%d ", i);
        }
    }
    return 0;
}

File: PRAK401-2310817220016-NailaHanifah.py
def main():
    kelipatan = int(input("Input pertama (bilangan kelipatan): "))
    simbol = input("Input kedua (simbol pengganti): ")

    for i in range(1, 51):
        if i % kelipatan == 0:
            print(simbol, end=" ")
        else:
            print(i, end=" ")

if __name__ == "__main__":
    main()